src/components/Pillar/Pillar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default function Pillar({
+  count,
+  label,
+}: {
+  count: string;
+  label: string;
+}) {
+  return (
+    <div className="flex max-w-xs md:max-w-sm space-x-4 p-4 border rounded-lg items-center md:bg-[#09AFC6]">
```

Comment:
On mobile, this should have a white background so we can be able to place it anywhere and it still matches the design:

<img width="188" alt="image" src="https://user-images.githubusercontent.com/5578688/197340903-811d6ae7-d980-4067-a3ae-51e0772bffff.png">

We should also give it a min-width on both mobile and desktop as well to prevent it from looking squashed if the label is short:

<img width="162" alt="image" src="https://user-images.githubusercontent.com/5578688/197341097-c9682b7a-cac7-489b-924f-311bb7e57d3f.png">

Reply:
@antosan thanks for pointing out the background color I had not noticed that.